debian/kubelet/kubelet-wrapper

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+source /etc/default/kubelet
```

Comment:
I think  /etc/default files are usually not env files? Is that not true? Lot's of stuff doesn't work in env files, e.g.
```
DOCKER_OPTS="${DOCKER_OPTS} --dns 8.8.8.8"
```
Wouldn't work because it requires a bash expansion